### PR TITLE
fix: darkmode doesn't stay on refresh #218

### PR DESCRIPTION
**Pull request by Lourvens**
Closes #

### What new changes did you make? Tick all applicable boxes
- [ ] Added new abbreviation
- [v] Fixed something in the source code
- [ ] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### CODE REVIEW

**Comment by Lourvens** [comment]
Closes #218 

### Describe the new changes you added.
Save the last theme on local storage and apply it to the webpage when loaded which automatically solves this issue.

### Share a screenshot of new changes
chrome-capture-2023-0-31 (1)

File: src/components/navbar.jsx [hunk 1]
```diff
@@ -2,40 +2,38 @@ import { useEffect } from "react";
...
       icon: "moon",
-      text: "light"
+      text: "light",
     },
     {
       icon: "sunny",
-      text: "dark"
-    }
+      text: "dark",
+    },
```
Comment (Njong392) [inline src/components/navbar.jsx:18]: I think there is a trailing comma here?
```suggestion
    }
```

File: src/components/navbar.jsx [hunk 2]
```diff
@@ -2,40 +2,38 @@ import { useEffect } from "react";
...
+  const defaultTheme = localStorage.getItem("theme") || "light";
+  const [theme, setTheme] = useState(defaultTheme);
   const element = document.documentElement;
-  
+
   const icons = [
     {
       icon: "moon",
-      text: "light"
+      text: "light",
```
Comment (Njong392) [inline src/components/navbar.jsx:13]: Trailing comma?
```suggestion
      text: "light"
```

File: src/components/navbar.jsx [hunk 3]
```diff
@@ -2,40 +2,38 @@ import { useEffect } from "react";
...
     },
     {
       icon: "sunny",
-      text: "dark"
-    }
+      text: "dark",
```
Comment (Njong392) [inline src/components/navbar.jsx:17]: I believe, there is another here. Am I the one missing something perhaps?
```suggestion
      text: "dark"
```

**Review by Njong392 — CHANGES_REQUESTED** [review]
Amazing changes you've made! Thank you very much 🎉 Please take a look at the changes I have suggested.

File: src/components/navbar.jsx (hunk 3 above)
Comment (Lourvens) [reply src/components/navbar.jsx:17]: Oh I see it's because my code formatter. eslint has format the code automatically

**Comment by Lourvens** [comment]
@Njong392  I made the changes you suggested, Think it's time to review it thanks

**Review by Njong392 — APPROVED** [review]
Looks all good now! Thank you so much for the stellar addition 🎉

**Pull request merged**